d2renderers/d2svg/d2svg.go

```diff
@@ -224,6 +227,42 @@ func arrowheadMarker(isTarget bool, id string, connection d2target.Connection) s
 				width*0.6, height*7/8,
 			)
 		}
+	case d2target.FilledCircleArrowhead:
+		attrs := fmt.Sprintf(`class="connection" fill="%s" stroke-width="%d"`, connection.Stroke, connection.StrokeWidth)
+		if isTarget {
+
+			path = fmt.Sprintf(`<circle %s clip-path="circle()" cx="%f" cy="%f" r="%f"/>`,
+				attrs,
+				(height/1.6)/strokeWidth*2,
+				(height/1.6)/strokeWidth*2,
+				(width/3.2)/strokeWidth*2,
+			)
+		} else {
+			path = fmt.Sprintf(`<circle %s clip-path="circle()" cx="%f" cy="%f" r="%f"/>`,
+				attrs,
+				(height/1.6)/strokeWidth*2,
+				(height/1.6)/strokeWidth*2,
+				(width/3.2)/strokeWidth*2,
+			)
```

Comment:
```suggestion
		attrs := fmt.Sprintf(`class="connection" fill="%s" stroke-width="%d"`, connection.Stroke, connection.StrokeWidth)
		radius := width/2
		if isTarget {

			path = fmt.Sprintf(`<circle %s cx="%f" cy="%f" r="%f"/>`,
				attrs,
				radius+strokeWidth/2,
				radius,
				radius-strokeWidth/2,
			)
		} else {
			path = fmt.Sprintf(`<circle %s cx="%f" cy="%f" r="%f"/>`,
				attrs,
				radius-strokeWidth/2,
				radius,
				radius-strokeWidth/2,
			)
```

I think this one should look like this, what is especially confusing is the x coordinate moves the arrowhead up or down then the connection is vertical. So for the target we need the +strokeWidth/2 to move down and the source needs the opposite.